cards/src/UserCard.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import getUsers from './getUsers';
+
+const useStyles = makeStyles({
+  root: {
+    maxHeight: 1000,
+    maxWidth: 545,
+  },
+});
+
+const UserCard = () => {
+  const [user, setUser] = React.useState({
+    avatar:'loading...',
+    email:'loading...',
+    first_name: 'loading...',
+    last_name:'loading...'
+  });
+let number = Math.floor(Math.random() * 12) + 1  
```

Comment:
This is pretty cool! It might be neat to store the number in state, and then give the user a button to update it. 

```js
const randomUserId  = () => Math.floor(Math.random() * 12) + 1  ;
const [userId, setUserId] = useState(randomUserId());

const handleClick = () => setUserId( randomUserId() );
```

Reply:
Actually, now that I think about it more, I'd probably include this logic in the parent `App` component, and then pass the `userId` as a prop to this component. That makes the user card a little bit more flexible.